bash-completion/setpriv

```diff
@@ -87,6 +87,16 @@ _setpriv_module()
 			COMPREPLY=( $(compgen -W "profile" -- $cur) )
 			return 0
 			;;
+		'--landlock-access')
+			# FIXME: how to list landlock accesses?
```

Comment:
Landlock does not currently provide an API to list all existing Landlock accesses with their names. You need to use the ones from the header.

Reply:
We can rely on the Landlock ABI version to infer the supported access rights, but we indeed need to have the name mapping. I don't think this is an issue because util-linux would need to know how to use the new access rights anyway, so if they are not part of the code there is no need to use them.